text/trace/0192-messaging-semantic-conventions-spec.md

```diff
@@ -0,0 +1,165 @@
+# Stabilizing messaging semantic conventions for tracing
+
+This document aims to describe the necessary changes for bringing the [existing semantic conventions for messaging](https://github.com/open-telemetry/opentelemetry-specification/blob/a1a8676a43dce6a4e447f65518aef8e98784306c/specification/trace/semantic_conventions/messaging.md)
+from the current [experimental](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/versioning-and-stability.md#experimental)
+to a [stable](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/versioning-and-stability.md#stable)
+state.
+
+This OTEP is based on [OTEP 0173](0173-messaging-semantic-conventions.md),
+which defines basic terms and describes messaging scenarios that should be
+supported by messaging semantic conventions.
+
+## Motivation
+
+This OTEP serves as a foundation for a first stable version of [messaging semantic conventions for tracing](https://github.com/open-telemetry/opentelemetry-specification/blob/a1a8676a43dce6a4e447f65518aef8e98784306c/specification/trace/semantic_conventions/messaging.md).
+It aims to define clear, consistent, and extensible conventions, it also
+describes reasons and motivations that led to the formulation of those
+conventions. The conventions comprise areas such as context propagation, span
+structure, names, and attributes.
+
+After this OTEP is merged, the changes it proposes will be merged into the
+messaging semantic conventions, which will subsequently be declared
+[stable](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/versioning-and-stability.md#stable).
+
+## Proposed stable messaging semantic conventions for tracing
+
+### Terminology
+
+The terminology used in this document is based on the [CloudEvents specification](https://github.com/cloudevents/spec/blob/v1.0.1/spec.md).
+CloudEvents is hosted by the CNCF and provides a specification for describing
+event data in common formats to provide interoperability across services,
+platforms and systems.
+
+#### Message
+
+A "message" is a transport envelope for the transfer of information. The
+information is a combination of a payload and metadata. Metadata can be
+directed at consumers or intermediaries on the message path. Messages are
```

Comment:
What do you mean by "directed at consumers or intermediaries on the message path"?

Reply:
It means that the metadata on a message can be used either by:
1. the consumer, or
2. intermediaries that process and forward messages.